packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [35.1.0]
+
+### Added
+
+- Add optional `topBid` property to the `NftMetadata` type. This property must be of type `TopBid`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add optional `floorAsk` property to the `TokenCollection` type. This property must be of type `FloorAskCollection`. ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Add linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
+- Fetch NFT collections data from the NFT-API `getCollections` API when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
+
+### Changed
+
+- Change attributes type in `NftMetadata` to `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Bump `@metamask/utils` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
+- Bump `@metamask/rpc-errors` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))
+
```

Comment:
In the Changelog section of the PR description, the `attributes` type change is prepended with "FIXED". Should this go under the `Fixed` header?

In addition, since `@metamask/assets-controllers` exports this type, should we mark this as a breaking change? 

``` suggestion
- Bump `@metamask/utils` to `^9.0.0` ([#4516](https://github.com/MetaMask/core/pull/4516))
- Bump `@metamask/rpc-errors` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))

### Fixed

- **BREAKING:** Change `attributes` type in `NftMetadata` to `Attributes[]` ([#4522](https://github.com/MetaMask/core/pull/4522))
```

Reply:
Thanks @mcmire for the review 🙏 
I checked how clients are using this attribute,
I see extension is not using it so far, i have an ongoing PR that incorporates the Attributes[];
Same goes for mobile;

So i thought maybe it is not necessary to mark it as breaking;

I could update it to Breaking if you think it is better!

Reply:
@sahar-fehri It's good to check whether the extension or mobile app is currently using this, so thanks for doing that. I think it would still be good practice, however, to mark this as breaking, so that if the clients encounter an unexpected error while upgrading in the near future or if we simply want to look back at history in the far future, we are doing our part to share that information now and aren't hiding it.

Reply:
That makes sense! i updated the version to `36.0.0`

Reply:
Thanks!